src/Hyperlink/Hyperlink.stories.jsx

```diff
@@ -16,31 +19,27 @@ const onClick = (event) => {
   action('HyperLink Click');
 };
 
+const wrapComponent = component => (
+  withInfo({ styles: FontAwesomeStyles }, README)(() => component)
+);
+
 storiesOf('HyperLink', module)
-  .add('minimal usage', () => (
-    <Hyperlink
-      destination="https://en.wikipedia.org/wiki/Hyperlink"
-      content="edX.org"
-    />
-  ))
-  .add('with blank target', () => (
-    <Hyperlink
-      destination="https://www.edx.org"
-      content="edX.org"
-      target="_blank"
-    />
-  ))
-  .add('with onClick', () => (
-    <Hyperlink
-      destination="https://www.edx.org"
-      content="edX.org"
-      target="_blank"
-      onClick={onClick}
-    />
-  ))
-  .add('with icon as content', () => (
-    <Hyperlink
-      destination="https://www.edx.org"
-      content={(<span className="fa fa-book" />)}
-    />
-  ));
+  .add('minimal usage', wrapComponent(<Hyperlink
```

Comment:
I'm not a big fan of this format, but there are `eslint` violations (`unexpected new line`) when `<Hyperlink />`  is on it's own line. 😞